### DISCUSSION TITLE
Checksum issues

### DISCUSSION THREAD

**Discussion by Boorinio** [livewire/livewire#5135] [7. Bugs (archived)] [answered]
Hello,
We have been using livewire in our project for sometime now and we noticed we were getting this kind of error in a specific component:
 Livewire\Exceptions\CorruptComponentPayloadExceptionPOST /livewire/message/backoffice.order.order-edit
Livewire encountered corrupt data when trying to hydrate the [backoffice.order.order-edit] component. Ensure that the [name, id, data] of the Livewire component wasn't tampered with between requests.

After further investigation we noticed that we were sending products into the frontend and some of them had extra spaces at the end. At the first request to the component the products were sent back from the frontend trimmed, resulting in a different checksum.

We use v.2.10.6

**Comment by joshhanley** [comment]
@Boorinio oh really? Yeah definitely shouldn't be doing that! Could you share some code which reproduces the issue with as little code as possible so we can investigate it?

Thanks!

**Reply by Boorinio** [reply]
Sorry for waisting your time, I was in the process of creating a demo project and found an old Middleware in our code base that sanitize the input. and cause the problem. We can close the discussion :)

**Reply by joshhanley** [reply]
@Boorinio ah no worries! Glad you sorted it 🙂 Can you copy your response to an answer below and then mark it as the answer? Thanks!

**Comment by Boorinio** [comment, marked as answer]
found an old Middleware in our code base that sanitize the input. and cause the problem
